app/models/asset_field_type.rb

```diff
@@ -30,6 +30,7 @@ def data
       'asset': {
         'file_name': asset_file_name,
         'url': asset.url,
+        'image': !!image?,
```

Comment:
To conform to Ruby conventions, rredicate methods must always return true or false - we shouldn't have to cast to a Boolean here